README.md

```diff
@@ -388,58 +397,54 @@ cd ~/eos/build/programs/eosd/
 <a name="walletimport"></a>
 ### Setting up a wallet and importing account key 
 
-Before running API commands you need to import the private key of an account you will be authorizing the transactions under into the EOS wallet.
-
-As you've previously added `plugin = eos::wallet_api_plugin` into `config.ini`, EOS wallet will be running as a part of `eosd` process.
-
-For testing purposes you can use a pre-created account `inita` from the `genesis.json` file.
-
-To login you need to run a command importing an active (not owner!) private key from `inita` account (you can find it in `~/eos/build/programs/eosd/data-dir/config.ini`) to the wallet.
+As you've previously added `plugin = eos::wallet_api_plugin` into `config.ini`, EOS wallet will be running as a part of `eosd` process. Every contract requires an associated account, so first, create a wallet.
 
 ```bash
 cd ~/eos/build/programs/eosc/
 ./eosc wallet create # Outputs a password that you need to save to be able to lock/unlock the wallet
-./eosc wallet import 5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3
 ```
 
-Now you can issue API commands under `inita` authority.
+For the purpose of this walkthrough, we'll import the private key of the `inita` account so that you can issue API commands under authority of an existing account. This private key can be found within your `config.ini` and is provided for testing purposes. 
```

Comment:
Should we say that 'inita' is a pre-created account from genesis file?

Reply:
That's not related to this issue, I'll commit to `docs-434` and merge into here.

Reply:
Additionally, there's actually instructions in here from before modification to create `inita` account that I noticed but didn't have time to address

Reply:
Ok, I see the problem. The tester account was initially used in this example to demonstrate creating a wallet,  but the convenience of the existing `inita` account was never removed. Then there was a mistake where `inita` was referenced for create wallet.